### ISSUE TITLE
 Wrong type used in unidecode in python2.

### ISSUE THREAD

**Issue by kazz-s** [mozilla/unicode-slugify#26]
https://github.com/mozilla/unicode-slugify/blob/c3b1f4dc3ff9ea706266e621f67977f74c966462/slugify/__init__.py#L74

Here if rv happens to be an empty list then type of new will be regular string not Unicode.
When called unidecode on it confusing warnings appears.

**Comment by nitely** [comment]
I don't think so. Try `print(type(new))`. There is a `from __future__ import unicode_literals` import, so the literal string there should be a unicode string.
